app/ApiClients/BingApi.php

```diff
@@ -34,7 +34,7 @@ class BingApi extends BaseApi implements Geocoder
 {
     use GeocodingHelper;
 
-    protected $base_uri = 'http://dev.virtualearth.net';
+    protected ?string $base_uri = 'http://dev.virtualearth.net';
     protected $geocoding_uri = '/REST/v1/Locations';
```

Comment:
While at it, could you typehint all of the extra properties?

Reply:
Sure, I was trying to achieve something simple first. Now it works, I'll do the same with the extra properties.